package.json

```diff
@@ -203,6 +203,7 @@
     "shlex": "2.1.0",
     "simple-git": "3.6.0",
     "slugify": "1.6.5",
+    "snappy": "7.1.1",
```

Comment:
can't we use some of our existing deps or node buildins? looks like a pretty huge dep and yarn will always install all optional depencies. 🤔